pkgs/by-name/gr/graplang/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchurl
+, flex
+, bison
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "graplang";
+  version = "1.46";
```

Comment:
```suggestion
  version = "1.46";
  
  outputs = [ "out" "man" ];
```

Reply:
Thanks for taking the time to review this PR and suggesting an improvement, @pbsds, much appreciated.